lua/lint/linters/golangcilint.lua

```diff
@@ -43,7 +43,7 @@ return {
           },
         },
         severity = sv,
-        message = item.Text,
+        message = item.FromLinter .. ": " .. item.Text,
```

Comment:
What do you think of setting the `source` property instead? 

That would have the advantage that users can configure whether to display it as part of their `vim.diagnostic` configuration (at least in neovim 0.6). See the `source`, `prefix` and `format` options in the `vim.diagnostic` help

Reply:
That makes more sense, and I wasn't familiar enough with the API to spot that solution myself. Updated 🙂